experimental/packages/otlp-exporter-base/src/OTLPExporterBase.ts

```diff
@@ -110,8 +110,14 @@ export abstract class OTLPExporterBase<
     return this._shutdownOnce.call();
   }
 
+  /**
+   * Exports any pending spans in the exporter
+   */
   forceFlush(): Promise<void> {
-    return Promise.resolve();
+    return Promise.all(this._sendingPromises)
+      .then(() => {
+        /** ignore resolved values */
+      });
```

Comment:
No need to do `.then` if you're just going to ignore the value.

Reply:
I can remove this, it was mostly a refactor (I moved the code out of the current shutdown function

Reply:
ah i see. i would just remove it since it doesn't serve a purpose

Reply:
Okay I will wrap this in a new promise, returning promise.all wouldn't work with the function type returning a promise<void>